odoo/service/model.py

```diff
@@ -35,15 +35,36 @@ def __str__(self):
         return ', '.join(params)
 
 
-def call_kw(model: BaseModel, name: str, args: list, kwargs: Mapping):
-    """ Invoke the given method ``name`` on the recordset ``model``. """
+def public_model_method(model: BaseModel, name: str):
```

Comment:
I suggest `get_public_method()`, and make it return the method from the class (instead of a bound method).  In other words:
```py
method = get_public_method(model, name)  # crashes if not public
result = method(records, *arg, **kwargs)
```